src/renderer/containers/LayersPanel.tsx

```diff
@@ -270,6 +307,18 @@ class LayersPanel extends React.Component<ILayersPanelProps & DispatchProp<State
         event.stopPropagation();
     }
 
+    private static renderSeparatorLineWithTitle(title: string) {
+        return (
+            <div style={{margin: '10px 0', padding: '5px', backgroundColor: '#293742'}}>
```

Comment:
Please improve performance and readability by extracting all inline styles as static const class variables. Try reusing these constants.